tests/test_inter_area_agent.py

```diff
@@ -83,6 +83,9 @@ def __init__(self, offers, bids=[], m_id=123, transfer_fees=transfer_fees, name=
     def sorted_offers(self):
         return list(sorted(self.offers.values(), key=lambda b: b.price / b.energy))
 
+    def get_bids(self):
+        return self.bids
```

Comment:
Can you use the deepcopy here to, please? Just to be consistent? Thanks!

Reply:
Will do, thanks